### PR TITLE
Readme update for sasToken API in builder

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ and
 - Renamed `setTier` to `setAccessTier` from `BlobAsyncClientBase` and `BlobClientBase` classes.
 - Added `ParallelTransferOptions` to buffered upload, upload from file and download to file methods.
 - Removed `Metadata` class and uses Map<String, String> for `matadata` field of `BlobProperties` and `ContainerProperties`.
+- Removed generate SAS token APIs.
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md]:
```suggestion
- Removed SAS token generation methods for client APIs, use {insert SAS token generation classes here} to create SAS tokens.
```

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,8 @@ and
...
+- Added `sasToken(String)` in client builder.
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md]:
```suggestion
- Replaces `credential(SASTokenCredential)` with `sasToken(String)` in client builder.
```

Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md]: Add a comment about removing `SASTokenCredential` and `SASTokenCredentialPolicy`

**Comment by sima-zhu** [comment]
Comments addressed

**Pull request merged**